test/tests/divertor_monoblock/divertor_monoblock.i

```diff
@@ -683,18 +695,24 @@ tungsten_atomic_density = ${units 6.338e28 m^-3}
 [Functions]
     ### Maximum mobile flux of 7.90e-13 at the top surface (1.0e-4 [m])
     ### 1.80e23/m^2-s = (5.0e23/m^2-s) *(1-0.999) = (7.90e-13)*(${tungsten_atomic_density})/(1.0e-4)  at steady state
+    [plasma_time_func]
+        type = ParsedFunction
+        expression =   'if((t % ${plasma_cycle_time}) < ${plasma_ramp_time}, (t % ${plasma_cycle_time})/${plasma_ramp_time},
+                        if((t % ${plasma_cycle_time}) < ${plasma_ss_end_time}, 1,
+                        if((t % ${plasma_cycle_time}) < ${plasma_ramp_down_time}, 1 - ((t % ${plasma_cycle_time})-${plasma_ss_end_time})/${plasma_ramp_time}, 0.0)))'
+    []
     [mobile_flux_bc_func]
         type = ParsedFunction
-        expression =   'if((t % 1600) < 100.0, 0.0      + 7.90e-13*(t % 1600)/100,
-                        if((t % 1600) < 500.0, 7.90e-13,
-                        if((t % 1600) < 600.0, 7.90e-13 - 7.90e-13*((t % 1600)-500)/100, 0.0)))'
+        symbol_values = 'plasma_time_func ${plasma_max_flux} ${plasma_min_flux}'
+        symbol_names = 'time_func max_val min_val'
+        expression = '(max_val - min_val) * time_func + min_val'
```

Comment:
Wonder if this makes it simpler to track which function it is? (Assuming it works?)
```suggestion
        symbol_values = 'plasma_time_func'
        symbol_names = 'time_func'
        expression = '(${plasma_max_flux} - ${plasma_min_flux}) * time_func + ${plasma_min_flux}'
```
... and similarly for other cases

Reply:
Yes, this removes unnecessary intermediate steps/quantities/definitions.